README.md

```diff
@@ -112,7 +112,7 @@ elm-review init --template jfmengels/elm-review-config/package
 ```
 
 Once you have set up an initial configuration, you can add new rules. As `elm-review` does not
-[come with built-in rules](https://github.com/jfmengels/elm-review/blob/main/documentation/design/no-built-in-rules.md),
+[come with built-in rules](https://github.com/jfmengels/elm-review/blob/master/documentation/design/no-built-in-rules.md),
```

Comment:
> As a random human, do you mind deleting node-elm-review:master?
It's confusing. As is using elm-review:master rather than main.

Yes, I can do that. But then this link should not get updated, correct?